mkdocs.yml

```
@@ -36,6 +37,7 @@ nav:
     - Uninstall an Extension: tutorials/uninstall-extension.md
   - How-To Guides:
     - Catalog queries: howto/catalog-queries.md
+    - Configure Global pull secrets: howto/configure-global-pull-secrets.md
```

```suggestion
    - Configure global pull secrets to pull from private registries: howto/configure-global-pull-secrets.md
```
maybe a little more informative wdyt?